smartsim/log.py

```diff
@@ -125,5 +125,7 @@ def log_to_file(filename: str, log_level: str = "debug") -> None:
     :type log_level: int | str
     """
     logger = logging.getLogger("SmartSim")
-    stream = open(filename, "w+", encoding="utf-8")  # pylint: disable=consider-using-with
+    stream = open(
+        filename, "w+", encoding="utf-8"
+    )  # pylint: disable=consider-using-with
```

Comment:
Per `pylint` output
```suggestion
    stream = open(  # pylint: disable=consider-using-with
        filename, "w+", encoding="utf-8"
    )
```

Reply:
I saw this comment appear in real time and it scared me.